external-docs/docs/guides/configuration/qube-config/columns/measures.md

```diff
@@ -0,0 +1,143 @@
+
+Note that if you use one of the [conventional column titles](TODO) for measures then the above configuration is
+equivalent to what csvcubed would do to your column by default.
+
+<!-- ```json
+{
+    "$schema": "https://purl.org/csv-cubed/qube-config/v1",
+    "title": "Average Height and Weight for Men in different countries",
+    "columns": {
+        "Measure": {
+            "type": "measures",
+            "values": [
+              {
+                "Measure 1": {
+                  "label": "Average Height",
+                  "description": "The average height for men in the observed country."
+                },
+                "Measure 2": {
+                  "label": "Average Weight",
+                  "description": "The average weight for men in the observed country."
+                }
+              }
+          ]
+        },
+    }
+}
+``` -->
+
+One of the advantages of measure columns in standard shape data sets is that no changes are required in the cube
+configuration file if new measures are added. Using multiple measures in a measure column simply means
+adding new rows to the data set, and specifying measures (and units) to be used for the observation.
+
+To view more information on the difference between single measure and multi measure data sets, see the
+[Shape your data](../../../shape-data/index.md) page (for both standard and pivoted shape).
+
+### Optional properties
+
+When defining a measures column, there are optional properties that can be entered, depending on how your measures are
+being defined within the column.
+
+If you are creating new measures within your measures column, the details of the new measures should be entered into a
+`values` field. The JSON below shows an example of the `values` field used in a measures column.
+
+```json
+{
+    "$schema": "https://purl.org/csv-cubed/qube-config/v1",
+    "title": "Average Height and Weight for Men in different countries",
+    "columns": {
+        "Measure": {
+            "type": "measures",
+            "values": true
+        },
+    }
+}
+```
+
+By default, the `values` field is set to `true`. This indicates to csvcubed to automatically generate
+[measure definitions](../measure-definitions.md) unique to your data set. See the previous link for more information
+on configuring measures and the fields that can be provided to the `values` object list.
+
+If you are re-using existing measures in your measures column, then do not use the `values` field to define the
+measures. Instead, use the field `cell_uri_template` to define your existing measure. The JSON below shows an example
+of this field in use.
+
+```json
+{
+    "$schema": "https://purl.org/csv-cubed/qube-config/v1",
+    "title": "Average Height and Weight for Men in different countries",
+    "columns": {
+        "Measure": {
+            "type": "measures",
+            "cell_uri_template": "http://example.org/measures/example-measure"
```

Comment:
I'd expect to see a variable inside this `cell_uri_template` referring back to the csvw_name_safed column name.